src/workloads/query/multiplanner/Simple.yml

```diff
@@ -155,6 +153,74 @@ Actors:
       database: *db
       collection: *coll
       selectivity: *selectivity
```

Comment:
This is no longer relevant, right? Since you have extracted the query from the template and are supplying the selectivity value directly? I also think that the predicate on x1 will default to 1 in this case. We can now directly use selectivity.

Reply:
Thanks for catching that!